views/js/qtiCreator/widgets/helpers/deletingState.js

```diff
@@ -13,99 +13,97 @@
+        $('body').on('mousedown.deleting keydown.deleting', function (e) {
+            if (e.ctrlKey || e.metaKey) {
                 //trigger undo callback if the standard keyboard shortcut ctrl+z is triggered
-                if(e.keyCode == 90){//z-key
+                if (e.keyCode === 90) {
+                    //z-key
                     undo($messageBox);
                 }
                 e.preventDefault();
                 return;
             }
 
             //confirm deleting whenever user interact with another object
-            if(e.target !== $messageBox[0] && !$.contains($messageBox[0], e.target)){
+            if (e.target !== $messageBox[0] && !$.contains($messageBox[0], e.target)) {
                 _confirmDeletion($messageBox, 400);
             }
         });
-        
-        $messageBox.find('a.undo').on('click', function(e){
+
+        $messageBox.find('a.undo').on('click', function (e) {
             e.preventDefault();
             undo($messageBox);
         });
 
-        $messageBox.find('.close-trigger').on('click', function(e){
+        $messageBox.find('.close-trigger').on('click', function (e) {
             e.preventDefault();
             _confirmDeletion($messageBox, 0);
         });
 
-        setTimeout(function(){
+        setTimeout(function () {
             _confirmDeletion($messageBox, 1000);
         }, _timeout);
     };
 
-    var _confirmDeletion = function($messageBox, fadeDelay){
-        //only allow deletion if the message has not already been deleted yet
-        if($messageBox.length && $.contains(document, $messageBox[0])){
-            $messageBox.trigger('confirm.deleting');
-            $messageBox.fadeOut(fadeDelay, function(){
-                _destroy($messageBox);
-            });
-        }
-    };
-    
-    var deletingHelper = {
-        createInfoBox : function(widgets){
-
-            var $messageBox = $(deletingInfoTpl({
-                serial : 'widgets',
-                count : widgets.length
-            }));
+    const deletingHelper = {
+        createInfoBox: function (widgets) {
+            const $messageBox = $(
+                deletingInfoTpl({
+                    serial: 'widgets',
+                    count: widgets.length
+                })
+            );
 
             $('body').append($messageBox);
 
             $messageBox.css({
-                'display' : 'block',
-                'position' : 'fixed',
-                'top' : '50px',
-                'left' : '50%',
-                'margin-left' : '-200px',
-                'width' : '400px',
-                zIndex : 999999
+                display: 'block',
+                position: 'fixed',
+                top: '50px',
+                left: '50%',
+                'margin-left': '-200px',
+                width: '400px',
+                zIndex: 999999
             });
-            
+
             _bindEvents($messageBox);
-            
-            _.each(widgets, function(w){
```

Comment:
this condition is useless and make issue when user delete row/column in table contained inlineChoices and then undo this deletion